rudder-core/src/main/resources/reportsSchema.sql

```diff
@@ -140,7 +141,6 @@ CREATE SEQUENCE ruleSerialId START 1;
 -- that sequence is used for nodeJoinKey value
 CREATE SEQUENCE ruleVersionId START 1;
 
-
 -- Create the table for the reports information
 CREATE TABLE expectedReports (
```

Comment:
Yes clearly. It will go in a following PR where I'm adressing tests and cleaning code.